README.md

```diff
@@ -124,7 +124,7 @@ ExRabbitPool.PoolSupervisor.start_link(
 )
 ```
 
-### Setting up a direct exchange with multiple bindings
+### Setting up a direct exchange with the same bindings
```

Comment:
that wasn't a typo, extracted from [RabbitMQ Tutorial](https://www.rabbitmq.com/tutorials/tutorial-four-python.html)